### PR TITLE
DT-29706 Adding new Modal for PDF download alert instructions

### PR DESCRIPTION
**Pull request by zacharymorel**
## Description
Adding a modal after our user research session showed the other 3 during testing failed.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#29706

## Testing done
Ran and updated unit tests, ran e2e tests, ran the feature locally

## Screenshots
Screen Shot 2021-09-28 at 2 58 10 PM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/find-forms/containers/SearchResults.jsx [hunk 1]
```diff
@@ -224,6 +227,57 @@ export const SearchResults = ({
...
+          title="Adobe Reader DC Required"
+          visible={isOpen}
+        >
+          <>
+            <p className="vads-u-display--block vads-u-margin-bottom--3">
+              <span>
+                All PDF forms do not function fully in a web browser or other
+                PDF viewer. Please download the form and use Adobe Acrobat
+                Reader DC to fill out. For specific instructions about working
+                with PDF’s
```
Comment (Rleahy22) [inline src/applications/find-forms/containers/SearchResults.jsx:254]: Should this be `PDFs`?

File: src/applications/find-forms/containers/SearchResults.jsx [hunk 2]
```diff
@@ -224,6 +227,57 @@ export const SearchResults = ({
...
+              </span>{' '}
+              <a href="https://www.va.gov/resources/how-to-download-and-open-a-vagov-pdf-form">
+                please read out Resources and Support Article
+              </a>
+            </p>
+            <a
+              href="https://get.adobe.com/reader/"
```
Comment (Rleahy22) [inline src/applications/find-forms/containers/SearchResults.jsx:261]: I think we probably want `rel="noopener noreferrer"` on links to external domains.

**Review by Rleahy22 — APPROVED** [review]
Just 2 small comments but otherwise looks good

File: src/applications/find-forms/containers/SearchResults.jsx (hunk 1 above)
Comment (zacharymorel) [reply src/applications/find-forms/containers/SearchResults.jsx:254]: No idea, I got it from content.

File: src/applications/find-forms/containers/SearchResults.jsx (hunk 2 above)
Comment (zacharymorel) [reply src/applications/find-forms/containers/SearchResults.jsx:261]: Good catch brother!

File: src/applications/find-forms/containers/SearchResults.jsx (hunk 1 above)
Comment (zacharymorel) [reply src/applications/find-forms/containers/SearchResults.jsx:254]: I checked back with them, looks like I can change it, thanks for catching that.

**Pull request merged**